kernel/data-pipeline/cdc/core/src/main/java/org/apache/shardingsphere/data/pipeline/cdc/api/impl/CDCJobAPI.java

```diff
@@ -218,6 +220,45 @@ protected String marshalJobIdLeftPart(final PipelineJobId pipelineJobId) {
         return DigestUtils.md5Hex(text.getBytes(StandardCharsets.UTF_8));
     }
     
+    /**
+     * Start job.
+     *
+     * @param jobId             job id
+     * @param importerConnector importer connector
+     */
+    public void startJob(final String jobId, final ImporterConnector importerConnector) {
+        CDCJob job = new CDCJob(importerConnector);
+        PipelineJobCenter.addJob(jobId, job);
+        JobConfigurationPOJO jobConfigPOJO = getElasticJobConfigPOJO(jobId);
+        updateJobConfigurationDisabled(jobConfigPOJO, false);
+        OneOffJobBootstrap oneOffJobBootstrap = new OneOffJobBootstrap(PipelineAPIFactory.getRegistryCenter(PipelineJobIdUtils.parseContextKey(jobId)), job, jobConfigPOJO.toJobConfiguration());
+        job.setJobBootstrap(oneOffJobBootstrap);
+        oneOffJobBootstrap.execute();
+    }
+    
+    /**
+     * Update job configuration disabled.
+     *
+     * @param jobId    job id
+     * @param disabled disabled
+     */
+    public void updateJobConfigurationDisabled(final String jobId, final boolean disabled) {
+        JobConfigurationPOJO jobConfigPOJO = getElasticJobConfigPOJO(jobId);
+        updateJobConfigurationDisabled(jobConfigPOJO, disabled);
+    }
+    
+    private void updateJobConfigurationDisabled(final JobConfigurationPOJO jobConfigPOJO, final boolean disabled) {
+        // TODO, ensure that there is only one consumer at a time, job config disable may not be updated when the program is forced to close
+        jobConfigPOJO.setDisabled(disabled);
```

Comment:
Could this TODO resolved this time? What's the plan if it's not

Reply:
I will removed